src/utils/particle/bubbler.ts

```diff
@@ -90,27 +95,36 @@ export class Bubbler {
         const particle = this.particle;
 
         /* on click event */
-        let dx_mouse = particle.position.x - (container.interactivity.mouse.click_pos_x || 0);
-        let dy_mouse = particle.position.y - (container.interactivity.mouse.click_pos_y || 0);
-        let dist_mouse = Math.sqrt(dx_mouse * dx_mouse + dy_mouse * dy_mouse);
-        let time_spent = (new Date().getTime() - (container.interactivity.mouse.click_time || 0)) / 1000;
+        const dx_mouse = particle.position.x - (container.interactivity.mouse.click_pos_x || 0);
+        const dy_mouse = particle.position.y - (container.interactivity.mouse.click_pos_y || 0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)